ale/drivers/messenger_drivers.py

```diff
@@ -164,3 +172,13 @@ def starting_ephemeris_time(self):
             sclock = self.label['IsisCube']['Archive']['SpacecraftClockStartCount']
             self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
         return self._starting_ephemeris_time
+
+    @property
+    def optical_distortion(self):
+        return {
+            "messengermdis": {
```

Comment:
I'd rather this stay as transverse in the usgscsm ISD so as to not break compatibility between ALE and usgscsm. This will break existing ISDs if we update usgscsm to look for this. I'd rather make breaking changes like that together.